packaging/nnstreamer.spec

```diff
@@ -4,6 +4,11 @@
 %define		nnstexampledir	/usr/lib/nnstreamer/bin
 %define		tensorflow_support	0
 %define		enable_nnfw	1
+%define		armnn_support 0
+
+%ifnarch %arm aarch64
+%define armnn_support 0
```

Comment:
Then, who enables `armnn_support`?

Reply:
Currently `armnn_support` is disabled as it is not available via OBS. Once it is available, this will be enabled.

Reply:
OK